src/platform/ESP32/CHIPDevicePlatformConfig.h

```diff
@@ -78,3 +78,5 @@
 #define CHIP_DEVICE_CONFIG_EVENT_LOGGING_INFO_BUFFER_SIZE CONFIG_EVENT_LOGGING_INFO_BUFFER_SIZE
 #define CHIP_DEVICE_CONFIG_EVENT_LOGGING_DEBUG_BUFFER_SIZE CONFIG_EVENT_LOGGING_DEBUG_BUFFER_SIZE
 #define CHIP_DEVICE_CONFIG_LOG_PROVISIONING_HASH CONFIG_LOG_PROVISIONING_HASH
+#define CHIP_DEVICE_CONFIG_RANDOM_MAC CONFIG_WIFI_AP_RANDOM_MAC
+#define CHIP_DEVICE_RENDEZVOUS_MODE_SOFTAP (CONFIG_RENDEZVOUS_MODE == 1)
```

Comment:
@sweetymhaiske With this change, how will it work for `examples/temperature-measurement-app/esp32/` that does not have `CONFIG_RENDEZVOUS_MODE`?